Source/WebKit/Shared/EntryPointUtilities/Cocoa/XPCService/XPCServiceEntryPoint.h

```diff
@@ -118,6 +118,12 @@ void XPCServiceInitializer(OSObjectPtr<xpc_connection_t> connection, xpc_object_
             JSC::Options::useSharedArrayBuffer() = true;
             optionsChanged = true;
         }
+        if (xpc_dictionary_get_bool(initializerMessage, "disable-jit-cage")) {
```

Comment:
It would be nice to have a FIXME/radar to remove this in the future.

Reply:
Sounds good — adding a FIXME comment here and filed a https://bugs.webkit.org/show_bug.cgi?id=276012.

Thanks for the review!